ddtrace/ext/ci.py

```diff
@@ -114,6 +114,12 @@ def tags(env=None, cwd=None):
     # Tags provided by the user take precedence over everything
     tags.update({k: v for k, v in user_specified_git_info.items() if v})
 
+    branch = tags.get(git.BRANCH) or ""
+    if "tags/" in branch:
+        if not tags.get(git.TAG):
+            tags[git.TAG] = branch
+        del tags[git.BRANCH]
```

Comment:
It's a bit confusing to read that we delete the tag here and then appear to set it below (line 124). Although if I follow this correctly, we do `git.normalize_ref(None)` which returns `None` so we end up setting `tags[git.BRANCH]` to `None`. Could we move line 124 up into an `else` branch here to be more explicit?

Reply:
Fixed

Reply:
thanks! this, + the comment is much easier to read! 🙇